Library/src/Systems/Sequence/SequenceSystem.cpp

```diff
@@ -97,6 +135,16 @@ void SequenceSystem::GetSequencesByCriteria(const Array<String>& InSequenceKeys,
 											const csp::common::Map<csp::common::String, csp::common::String>& MetaData,
 											SequencesResultCallback Callback)
 {
+	for (size_t i = 0; i < InSequenceKeys.Size(); i++)
+	{
+		if (!ValidateKey(InSequenceKeys[i]))
+		{
+			CSP_LOG_FORMAT(csp::systems::LogLevel::Error, "Cannot get Sequence. New Key: %s contains invalid characters", InSequenceKeys[i].c_str());
```

Comment:
I don't think the log message should say "New Key:", just "Key:" as it is not a new key.